src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,45 @@
+.clock {
+  position: relative;
+  @include square($clockSize);
+  border: 3px solid black;
+  margin: 5% auto;
+  justify-content: center;
+  align-items: center;
+  background-color: #efd6d6;
```

Comment:
8'th point in checklist

Reply:
I understand that, but in that case idk what in that properties  i should move into another class